beacon-chain/node/node.go

```diff
@@ -85,11 +100,33 @@ func (b *BeaconNode) Close() {
 	close(b.stop)
 }
 
-func (b *BeaconNode) registerWeb3Service() error {
-	endpoint := b.ctx.GlobalString(types.Web3ProviderFlag.Name)
+func (b *BeaconNode) registerBeaconDB(path string) error {
+	config := &database.BeaconDBConfig{DataDir: path, Name: beaconChainDBName, InMemory: false}
+	beaconDB, err := database.NewBeaconDB(context.TODO(), config)
```

Comment:
Why not use `context.Background()` instead of `context.TODO()` . Same below

Reply:
@nisdas we’re using TODO for now until we have a definitive answer to what the parent context should be for these services.